src/main.js

```diff
@@ -1,4 +1,6 @@
 import { createApp } from 'vue'
 import App from './App.vue'
+import router from './router'
 
 createApp(App).mount('#app')
```

Comment:
const app = createApp(App)
and then app.use()

Reply:
Sorted (at least I think)